src/courtroom.cpp

```diff
@@ -2497,6 +2501,7 @@ void Courtroom::handle_emote_mod(int emote_mod, bool p_immediate)
     // If immediate is not ticked on...
     if (!p_immediate)
     {
+      play_sfx();
```

Comment:
Not understanding this line in particular but I'll let it pass

Reply:
Makes it so you don't have to have pre on for narrator chat but I confess I didn't test that use case

Reply:
This is what makes it function so it always plays sfx regardless of pre, if such sfx exists. This is what makes the feature work basically

Reply:
But for older clients doesn't the client put in the SFX into the emote, but sets the emote mod such that it doesn't play the SFX?

Reply:
wat

Reply:
> But for older clients doesn't the client put in the SFX into the emote, but sets the emote mod such that it doesn't play the SFX?

Wait...  Older clients.... 
😱

Reply:
@Crystalwarrior do you remember what our resolution was on this? I frankly forgot